embassy-stm32/src/sdmmc/mod.rs

```diff
@@ -227,7 +227,11 @@ const DMA_TRANSFER_OPTIONS: crate::dma::TransferOptions = crate::dma::TransferOp
     fifo_threshold: Some(crate::dma::FifoThreshold::Full),
 };
 #[cfg(all(sdmmc_v1, not(dma)))]
-const DMA_TRANSFER_OPTIONS: crate::dma::TransferOptions = crate::dma::TransferOptions {};
+const DMA_TRANSFER_OPTIONS: crate::dma::TransferOptions = crate::dma::TransferOptions {
+    circular: false,
```

Comment:
it'd be better to do `..Default::default()`, so this code doesn't need updating every time we add fields to TransferOptions.

Reply:
```
error[E0015]: cannot call non-const fn `<dma::bdma::TransferOptions as core::default::Default>::default` in constants
   --> src/sdmmc/mod.rs:234:7
    |
234 |     ..Default::default()
    |       ^^^^^^^^^^^^^^^^^^
    |
    = note: calls in constants are limited to constant functions, tuple structs and tuple variants
    = help: add `#![feature(const_trait_impl)]` to the crate attributes to enable
 ```
 
 Doesn't seem to be possible right now. That's probably why it is not done 2 lines before for the DMA options.

Reply:
oh okay :( 
whatever then